api.js

```diff
@@ -12,6 +14,16 @@ function getArticleById(articleId) {
 		});
 }
 
+function getArticlesByTopic(topic) {
+	return axios
+		.get('https://tobbit.onrender.com/api/articles', {
+			params: { topic, limit: 100 },
+		})
+		.then((response) => {
+			return response;
```

Comment:
try to return the most relevant thing here - we dont need the whole response object in the component, just the articles :)